rfd/cspell.json

```diff
@@ -3,98 +3,322 @@
   "language": "en",
   "words": [
     "AAAX",
-    "aabbcc",
     "ABCDEFGHIJKL",
     "ABCDEFGHIJKLMNOP",
     "ABRBRV",
-    "accid",
+    "ALPN",
+    "APIMFA",
+    "APNG",
+    "AQAB",
+    "ASDFGHJK",
```

Comment:
cSpell supports comments to allow specific works on specific files. For example, with yaml, you can write:

```yaml
# cspell:words someContextSpecificWordOrAcronym
```

Do all of these need to be added globally, or should some be allowed locally?

Reply:
Ah fair enough - I didn't catch that part.